R/constructNWISURL.r

```diff
@@ -244,8 +244,8 @@ constructWQPURL <- function(siteNumber,parameterCd,startDate,endDate){
     suppressWarnings(pCodeLogic <- all(!is.na(as.numeric(parameterCd))))
   } else {
     pCodeLogic <- FALSE
-    parameterCd <- gsub(",","%2C",parameterCd)
-    parameterCd <- URLencode(parameterCd)
+    # parameterCd <- gsub(",","%2C",parameterCd)
```

Comment:
Actually, probably a bunch of commented out stuff I need to delete now that you mention it.